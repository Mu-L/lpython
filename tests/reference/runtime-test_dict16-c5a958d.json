{
    "basename": "runtime-test_dict16-c5a958d",
    "cmd": "lpython {infile}",
    "infile": "tests/errors/test_dict16.py",
    "infile_hash": "7b00cfd7f6eac8338897bd99e5d953605f16927ee0f27683146b0182",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "runtime-test_dict16-c5a958d.stderr",
    "stderr_hash": "9581abf156cd7de65fb9e7dde107233b52c523eb106b90233a36c5a0",
    "returncode": -6
}