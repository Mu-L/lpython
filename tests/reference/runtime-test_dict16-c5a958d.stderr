Assertion failed: (Ty && "Invalid GetElementPtrInst indices for type!"), function checkGEPType, file Instructions.h, line 897.
Traceback (most recent call last):
[2m  File "[0m[1m[35m$DIR/lpython.cpp[39m[0m[2m", line 2248[0m
[2m  File "[0m[1m[35m$DIR/lpython.cpp[39m[0m[2m", line 1121[0m
[2m  File "[0m[1m[35m$DIR/python_evaluator.cpp[39m[0m[2m", line 464[0m
[2m  File "[0m[1m[35m$DIR/src/libasr/codegen/asr_to_llvm.cpp[39m[0m[2m", line 11939[0m
[2m  File "[0m[1m[35m$DIR/src/libasr/asr_base_visitor.h[39m[0m[2m", line 343[0m
[2m  File "[0m[1m[35m$DIR/src/libasr/asr_base_visitor.h[39m[0m[2m", line 318[0m
[2m  File "[0m[1m[35m$DIR/src/libasr/asr_base_visitor.h[39m[0m[2m", line 344[0m
[2m  File "[0m[1m[35m$DIR/src/libasr/asr_base_visitor.h[39m[0m[2m", line 22[0m
[2m  File "[0m[1m[35m$DIR/src/libasr/codegen/asr_to_llvm.cpp[39m[0m[2m", line 1019[0m
[2m  File "[0m[1m[35m$DIR/src/libasr/asr_base_visitor.h[39m[0m[2m", line 346[0m
[2m  File "[0m[1m[35m$DIR/src/libasr/asr_base_visitor.h[39m[0m[2m", line 30[0m
[2m  File "[0m[1m[35m$DIR/src/libasr/codegen/asr_to_llvm.cpp[39m[0m[2m", line 3517[0m
[2m  File "[0m[1m[35m$DIR/src/libasr/codegen/asr_to_llvm.cpp[39m[0m[2m", line 4871[0m
[2m  File "[0m[1m[35m$DIR/src/libasr/codegen/asr_to_llvm.cpp[39m[0m[2m", line 4498[0m
[2m  File "[0m[1m[35m$DIR/src/libasr/codegen/asr_to_llvm.cpp[39m[0m[2m", line 4824[0m
[2m  File "[0m[1m[35m$DIR/src/libasr/asr_base_visitor.h[39m[0m[2m", line 362[0m
[2m  File "[0m[1m[35m$DIR/src/libasr/asr_base_visitor.h[39m[0m[2m", line 54[0m
[2m  File "[0m[1m[35m$DIR/src/libasr/codegen/asr_to_llvm.cpp[39m[0m[2m", line 5675[0m
[2m  File "[0m[1m[35m$DIR/src/libasr/codegen/llvm_utils.cpp[39m[0m[2m", line 2882[0m
[2m  File "[0m[1m[35m$DIR/src/libasr/codegen/llvm_utils.cpp[39m[0m[2m", line 2441[0m
[2m  File "[0m[1m[35m$DIR/src/libasr/codegen/llvm_utils.cpp[39m[0m[2m", line 1504[0m
[2m  File "[0m[1m[35m$DIR/src/libasr/codegen/llvm_utils.cpp[39m[0m[2m", line 1664[0m
[2m  File "[0m[1m[35m/Users/swaminathshiju/miniconda3/envs/lp/include/llvm/IR/IRBuilder.h[39m[0m[2m", line 1773[0m
    return Insert(GetElementPtrInst::Create(Ty, Ptr, IdxList), Name);
[2m  File "[0m[1m[35m/Users/swaminathshiju/miniconda3/envs/lp/include/llvm/IR/Instructions.h[39m[0m[2m", line 943[0m
    NameStr, InsertBefore);
[2m  File "[0m[1m[35m/Users/swaminathshiju/miniconda3/envs/lp/include/llvm/IR/Instructions.h[39m[0m[2m", line 1151[0m
    }
[2m  File "[0m[1m[35m/Users/swaminathshiju/miniconda3/envs/lp/include/llvm/IR/Instructions.h[39m[0m[2m", line 1143[0m
    : Instruction(getGEPReturnType(PointeeType, Ptr, IdxList), GetElementPtr,
[2m  File "[0m[1m[35m/Users/swaminathshiju/miniconda3/envs/lp/include/llvm/IR/Instructions.h[39m[0m[2m", line 1074[0m
    Type *PtrTy = PointerType::get(checkGEPType(getIndexedType(ElTy, IdxList)),
[2m  File "[0m[1m[35m/Users/swaminathshiju/miniconda3/envs/lp/include/llvm/IR/Instructions.h[39m[0m[2m", line 898[0m
    return Ty;
[2m  Binary file "[0m[1m[35m/usr/lib/system/libsystem_c.dylib[39m[0m[2m", local address: 0x18035ac1b[0m
[2m  Binary file "[0m[1m[35m/usr/lib/system/libsystem_c.dylib[39m[0m[2m", local address: 0x18035b907[0m
[2m  Binary file "[0m[1m[35m/usr/lib/system/libsystem_pthread.dylib[39m[0m[2m", local address: 0x18044ef6f[0m
[2m  Binary file "[0m[1m[35m/usr/lib/system/libsystem_platform.dylib[39m[0m[2m", local address: 0x180484183[0m
Abort: Signal SIGABRT (abort) received

